django_mongodb/features.py

```diff
@@ -214,8 +215,8 @@ class DatabaseFeatures(BaseDatabaseFeatures):
             "timezones.tests.LegacyDatabaseTests.test_cursor_execute_accepts_naive_datetime",
             "timezones.tests.LegacyDatabaseTests.test_cursor_execute_returns_naive_datetime",
             "timezones.tests.LegacyDatabaseTests.test_raw_sql",
-            "timezones.tests.NewDatabaseTests.test_cursor_execute_accepts_aware_datetime",
-            "timezones.tests.NewDatabaseTests.test_cursor_execute_returns_aware_datetime",
+            "timezones.tests.NewDatabaseTests.test_cursor_execute_accepts_naive_datetime",
+            "timezones.tests.NewDatabaseTests.test_cursor_execute_returns_naive_datetime",
```

Comment:
I'm not sure I understand why the new test we skip is the naive version. So we will no longer support naive datetime instead?

Or is it that the naive tests were the original ones that should have been failing?

Reply:
Which tests run is controlled by `DatabaseFeatures.supports_timezones`, i.e. some tests are decorated with  `@skipUnlessDBFeature("supports_timezones")` or `@skipIfDBFeature("supports_timezones")`.

These are the tests that run when `supports_timezones = False` but we can't run them on MongoDB because they use SQL.